docs/porting-guide.rst

```diff
@@ -1,49 +1,311 @@
+Carbon Black Cloud product names have been updated in the SDK.
 
-The different API objects, ``CbDefenseAPI``, ``CbLiveQueryAPI``, ``CbPSCBaseAPI``, and ``CbThreatHunterAPI`` are
-replaced with ``CBCloudAPI``.  Import this object from the ``cbc_sdk`` package, i.e. ``from cbc_sdk import CBCloudAPI``.
++----------------------------+-------------------------------+
+| CBAPI Name (old)           | CBC SDK Name (new)            |
++============================+===============================+
+| ``cbapi.psc.defense``      | ``cbc_sdk.endpoint_standard`` |
++----------------------------+-------------------------------+
+| ``cbapi.psc.livequery``    | ``cbc_sdk.audit_remediation`` |
++----------------------------+-------------------------------+
+| ``cbapi.psc.threathunter`` | ``cbc_sdk.enterprise_edr``    |
++----------------------------+-------------------------------+
+| Does not exist in CBAPI    | ``cbc_sdk.platform``          |
++----------------------------+-------------------------------+
 
-**Enterprise EDR Process and Event Search replaced with Platform Search**
+Import statements will need to change:
 
-CBAPI used the ``threathunter`` (Enterprise EDR) package for Process and Event searches. Platform search is replacing these APIs,
-so the SDK now uses the ``platform`` package to handle Process and Event searches.
+::
 
-* ``threathunter.Process`` -> ``platform.Process``
-* ``threathunter.Event`` -> ``platform.Event``
+    # Endpoint Standard (Defense)
 
-Additionally, Process Summaries and Trees now use the same API route, with the addition of a ``?format=tree``
-query parameter to differentiate the two. Therefore, Process Trees have been integrated into the Process class.
+    # CBAPI
+    from cbapi.psc.defense import Device, Event, Policy
 
-* ``threathunter.Tree`` -> ``platform.Process.Tree``
+    # CBC SDK
+    from cbc_sdk.endpoint_standard import Device, Event, Policy
+
+::
+
+    # Audit and Remediation (LiveQuery)
+
+    # CBAPI
+    from cbapi.psc.livequery import Run, RunHistory, Result, DeviceSummary
+
+    # CBC SDK
+    from cbc_sdk.audit_remediation import Run, RunHistory, Result, DeviceSummary
+
+::
+
+    # Enterprise EDR (ThreatHunter)
+
+    # CBAPI
+    from cbapi.psc.threathunter import Feed, Report, Watchlist
+
+    # CBC SDK
+    from cbc_sdk.enterprise_edr import Feed, Report, Watchlist
+
+Moved Packages and Models
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Some modules have been moved to a more appropriate location.
+
++-----------------------------+------------------------------+
+| CBAPI Name (old)            | CBC SDK Name (new)           |
++=============================+==============================+
+| ``cbapi.example_helpers``   | ``cbc_sdk.helpers``          |
++-----------------------------+------------------------------+
+| ``cbapi.psc.alerts_query``  | ``cbc_sdk.platform.alerts``  |
++-----------------------------+------------------------------+
+| ``cbapi.psc.devices_query`` | ``cbc_sdk.platform.devices`` |
```

Comment:
Do we want to mention this format or should we be pushing them to be more consistent and use

```
from cbc_sdk.platform import Alerts, Devices
```

Reply:
I think the more consistent format `cbc_sdk.platform` is best, I've updated this